src/brsTypes/components/ComponentFactory.ts

```diff
@@ -18,6 +18,7 @@ import {
 
 export enum BrsComponentName {
     Node = "Node",
+    Node_lowerCase = "node",
```

Comment:
Is this a problem with _all_ components? What if I did `createObject("roSGNode", "group")` -- I'm assuming I'd encounter the same issue.

Also I think we should probably do this in a separate PR, just because it's not strictly related to `lookupCI`.

Reply:
Also, what about cases like `createObject("RoSGNode", "nOdE")`? Should those also create a Node object?

Reply:
yep, you are right, all naming for type of node is case insensitive (works for user components too).

Reply:
reverted fix for #638 issue